phpcs.xml

```diff
@@ -14,8 +14,8 @@
     <rule ref="Generic.Files.LineEndings" />
     <rule ref="Generic.Files.LineLength">
         <properties>
-            <property name="lineLimit" value="125" />
-            <property name="absoluteLineLimit" value="125" />
+            <property name="lineLimit" value="119" />
+            <property name="absoluteLineLimit" value="119" />
```

Comment:
Every other number but the current maximum of 119 would be arbitrary, and that's what I do not like and want to avoid, as explained. Discovering this is not totally obvious (the output will not tell you that you "should increase a number in the phpcs.xml file" or something), but it should not be that hard to understand what's going on. PHPCS output is pretty good. Much better than PHPMD, which doesn't even tell you the name of the rule you hit. :-(